tools/feature_alignment/feature_alignment.xml

```diff
@@ -1,168 +1,289 @@
+        --out out.tsv
         --out_matrix out_matrix.tsv
-        --out_ids $out_ids
-        --fdr_cutoff $fdr_cutoff
-        #if $max_fdr_quality:
-            --max_fdr_quality $max_fdr_quality
+        --out_ids out_ids.tsv
+        --file_format '$iformat'
+        --method '$method_sel.method'
+        --matrix_output_method '$matrix_output_method'
+        --realign_method '$realign_method'
+        #if $max_rt_diff_cond.max_rt_diff_sel == 'opt_max_rt_diff_enabled':
+            --max_rt_diff '$max_rt_diff_cond.max_rt_diff'
+        #end if
+        #if $iso_max_rt_diff_cond.iso_max_rt_diff_sel == 'opt_iso_max_rt_diff_enabled':
+            --iso_max_rt_diff '$iso_max_rt_diff_cond.iso_max_rt_diff'
+        #end if
+        #if $frac_selected_cond.frac_selected_sel == 'opt_frac_selected_enabled':
+            --frac_selected '$frac_selected_cond.frac_selected'
+        #end if
+        #if $fdr_cond.fdr_sel == 'opt_target_fdr':
+            --target_fdr '$fdr_cond.target_fdr'
+        #elif $fdr_cond.fdr_sel == 'opt_fdr_cutoff':
+            --fdr_cutoff '$fdr_cond.fdr_cutoff'
+        #else:
+            --target_fdr '0'
         #end if
-        #if $max_rt_diff:
-            --max_rt_diff $max_rt_diff
+        --max_fdr_quality '$max_fdr_quality'
+        #if $method_sel.method == 'LocalMST' or $method_sel.method == 'LocalMSTAllCluster':
+            --mst:useRTCorrection '$method_sel.mst_useRTCorrection'
+            --mst:Stdev_multiplier '$method_sel.mst_Stdev_multiplier'
+            --mst:useLocalStdev '$method_sel.mst_useLocalStdev'
+            --mst:useReference '$method_sel.mst_useReference'
         #end if
-        #if $iso_max_rt_diff:
-            --iso_max_rt_diff $iso_max_rt_diff
+        #if $further_params.disable_isotopic_grouping:
+            --disable_isotopic_grouping
         #end if
-        #if $frac_selected:
-            --frac_selected $frac_selected
+        #if $further_params.dscore_cond.dscore_sel == 'opt_dscore_enabled':
+            --use_dscore_filter
+            --dscore_cutoff '$further_params.dscore_cond.dscore_cutoff'
+        #end if
+        #if $further_params.nr_high_conf_exp_cond.nr_high_conf_exp_sel == 'opt_nr_high_conf_exp_enabled':
+            --nr_high_conf_exp '$further_params.nr_high_conf_exp_cond.nr_high_conf_exp'
+        #end if
+        #if $further_params.readmethod != 'disabled':
+            --readmethod '$further_params.readmethod'
+        #end if
+        #if $further_params.alignment_score_cond.alignment_score_sel == 'opt_alignment_score_enabled':
+            --alignment_score '$further_params.alignment_score_cond.alignment_score'
         #end if
-        --method $method
-        --matrix_output_method $matrix_output_method
-        --realign_method $realign_method
-        &&
-        mv out_matrix.tsv $out_matrix
 
-]]></command>
+    #if $iformat == 'openswath':
+        && cat out.tsv > '$out'
+    #else:
+        && echo '' > '$out'
```

Comment:
if this output is empty it should not be put into the history, left?

Reply:
Thanks for pointing that out!